utils/connect.js

```diff
@@ -1,6 +1,33 @@
-const { connect: nearConnect } = require('near-api-js');
+const { 
+    connect: nearConnect,
+    Account,
+    multisig: {
+        MULTISIG_CHANGE_METHODS
+    }
+} = require('near-api-js');
+const { AccountMultisig } = require('./2fa');
 
 module.exports = async function connect({ keyStore, ...options }) {
-    // TODO: Avoid need to wrap in deps
-    return await nearConnect({ ...options, deps: { keyStore }});
+    const near = await nearConnect({ ...options, deps: { keyStore }});
+    near.account = async (accountId) => {
+        const account = new Account(near.connection, accountId);
+        await account.state();
+        const accessKeys = await account.getAccessKeys();
+        const hasFAK = accessKeys.find((k) => k.access_key.permission && k.access_key.permission === 'FullAccess');
+        if (hasFAK) {
+            return account;
```

Comment:
shouldn't we check whether full access key actually exists locally? Otherwise returned `account` going to be unusable.

Reply:
e.g. this will happen for all seed phrase + 2FA accounts

Reply:
checking against
```
const localKey = (await near.connection.signer.getPublicKey(account.accountId, near.connection.networkId)).toString();
```